app/src/main/res/values/third_party_dependencies.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="third_party_dependency_name_0">androidx.annotation:annotation:1.1.0</string>
-  <string name="third_party_dependency_name_1">androidx.appcompat:appcompat:1.2.0</string>
-  <string name="third_party_dependency_name_2">androidx.arch.core:core-common:2.1.0</string>
-  <string name="third_party_dependency_name_3">"com.google.protobuf:protobuf-lite:3.0.0"</string>
+  <string name="third_party_dependency_name_0">androidx.annotation:annotation</string>
+  <string name="third_party_dependency_name_1">androidx.appcompat:appcompat</string>
+  <string name="third_party_dependency_name_2">androidx.arch.core:core-common</string>
+  <string name="third_party_dependency_name_3">"com.google.protobuf:protobuf-lite"</string>
```

Comment:
Why quotes? Won't that show up in the UI?

Reply:
Removed unnecessary quotes